client/extensions/woocommerce/style.scss

```diff
@@ -6,19 +6,13 @@
 }
 
 .woocommerce {
+	@import 'app/shipping/style';
```

Comment:
I think we need to decide exactly how we're using this `.woocommerce` class. IMO it should just be applied to the body tag on wc screens. At the moment we're adding it in a few different places which is causing some styling issues.

For example; trying to apply a width to the 'add zone' modal. We can't target it currently because it exists outside of any `.woocommerce` container.